src/main/java/jenkins/plugins/office365connector/Webhook.java

```diff
@@ -193,6 +210,42 @@ public String getDisplayName() {
             return "Webhook";
         }
 
+        public String getIp() {
+            return ip;
+        }
+
+        @DataBoundSetter
+        public void setIp(String ip) {
+            this.ip = ip;
```

Comment:
is it compatible with older version for which this attribute is not defined ?

Reply:
I don't understand this question.

Reply:
If i tested this correctly, it only becomes an issues when you rollback the plugin version to a version that doesn't include these changes. In this instance we can discard the unreadable data.

![image](https://user-images.githubusercontent.com/23258583/182949551-e8bbef11-1264-40c4-a765-c1b57ce481f8.png)